videostore/utils.h

```diff
@@ -0,0 +1,6 @@
+#ifndef UTILS_H
+#define UTILS_H
+#include <libavformat/avformat.h>
+int64_t get_video_duration(const char *filename);
+#define VIDEO_STORE_DURATION_RESP_ERROR -1
```

Comment:
have this be positive. That way we can return ffmpeg errors (which are all negative)

Reply:
also please define success

Reply:
Done.